action.yml

```diff
@@ -22,7 +22,7 @@ inputs:
     default: true
 outputs:
   python-version:
-    description: "The installed python version. Useful when given a version range as input."
+    description: "The installed Python version. Useful when given a version range as input."
```

Comment:
Python/PyPy

Reply:
Perhaps, let's rewrite descriptions for all inputs/outputs explicitly mentioning that they work for Python/PyPy?